code/modules/organs/subtypes/diona.dm

```diff
@@ -178,8 +178,22 @@
 	dislocated = -1
 	joint = "structural ligament"
 	amputation_point = "branch"
+	vital = FALSE // Lore team requested this, not vital organ. We can still live without it.
 
+/obj/item/organ/external/head/diona/New(loc, ...)
+	..()
+	if(owner)
+		owner.sdisabilities &= ~BLIND
+
+/obj/item/organ/external/head/diona/Destroy()
```

Comment:
What? No. Bad. Destroy should not have side-effects like this. remove() should always be run, this is unneeded.